lib/elrondex/transaction.ex

```diff
@@ -39,6 +39,15 @@ defmodule Elrondex.Transaction do
             # Computed signature based on 1-9 signature fields
             signature: nil
 
+  @doc """
+  Prepares a transaction
+
+  ## Arguments
+   * `account` - the account that makes the transaction
+   * `reciver` - the reciver
+   * `value` - value of transaction (hex)
+   * `data` - the data of transaction
```

Comment:
Data asociated to transaction, it is used in ESDT trasfer and Smart Contracts CALLs